sdk/keyvault/keyvault-secrets/.eslintrc.json

```diff
@@ -4,6 +4,8 @@
   "ignorePatterns": ["src/core"],
   "rules": {
     "@typescript-eslint/no-this-alias": "off",
-    "no-invalid-this": "off"
+    "no-invalid-this": "off",
+    "@azure/azure-sdk/ts-package-json-module": "warn",
+    "@azure/azure-sdk/ts-package-json-files-required": "warn"
```

Comment:
What line8-9 are for?

Reply:
It's because of changes like line 23 and line 28 here: https://github.com/Azure/azure-sdk-for-js/pull/8866/files#diff-9cb805b9c2b8f9e19a3d3937d6a28ce2R23-R28

I have ESLint enforced in these projects, and these rules would make it throw. If I run `rushx lint:fix`, then our plugins automatically overwrite my package.json and messes with the paths.

Reply:
Also cc: @willmtemple

Reply:
Seems like we need to fix these rules

Reply:
Yes we need to. The question is when! I think as part of another PR would be ideal.